tests/integration_toolbox.py

```diff
@@ -156,6 +158,46 @@ def testGetUrlToFileCachedKeepaliveChanged(self):
             conn.close()
         self.assertEqual(b'This is a test\n', data)
 
+    def testGetUrl404(self):
+        """Call get_url for a file that doesn't exist"""
+        with self.assertRaises(HTTPError):
+            data = spacepy.toolbox.get_url(
+                'http://localhost:{}/doesnotexist.txt'.format(self.port))
+    
+    def testGetUrlNoConnNoKeepAlive(self):
+        """Call get_url without specifying connection or keepalive"""
```

Comment:
This says without specifying a connection, but it specifies a connection.

Reply:
This was accidental, and it is now fixed, and the docstring is updated